evm/p2p/utils.py

```diff
@@ -0,0 +1,11 @@
+from evm.utils.numeric import safe_ord
+
+
+def ascii_chr(value):
+    return bytes([value])
+
+
+def sxor(s1, s2):
+    if len(s1) != len(s2):
+        raise ValueError("Cannot sxor strings of different length")
+    return b''.join(ascii_chr(safe_ord(a) ^ safe_ord(b)) for a, b in zip(s1, s2))
```

Comment:
I don't know if this is much better in terms of readability. but...

```python
bytes((itertools.starmap(operator.xor, zip(a, b))))
# or just
bytes(x ^ y for x, y in zip(a, b))
```

The `ascii_chr` and `safe_ord` are python2.7 compat right?  Those can be dropped?

Reply:
Yep, leftover from stuff I copied from pydevp2p